### PR TITLE
Draw text centered if X coordinate is left out

### PR DESCRIPTION
**Pull request by ErikDeBruijn**

### CODE REVIEW

File: Marlin/Marlin_main.cpp [hunk 1]
```diff
@@ -2323,20 +2323,30 @@ void process_commands()
     case 10000://M10000 - Clear the whole LCD
         lcd_lib_clear();
         break;
-    case 10001://M10001 - Draw text on LCD, M10002 X0 Y0 SText
+    case 10001://M10001 - Draw text on LCD, M10002 X0 Y0 SText (when X is left out, it will draw centered)
         {
-        uint8_t x = 0, y = 0;
```
Comment (oliv3r) [inline Marlin/Marlin_main.cpp]: why is there a code change on this line? I can't spot the difference :)

**Comment by ErikDeBruijn** [comment]
Right! I'll pay attention to that.

I tested it on a 2Go, seems to work fine. I'm fine with using this feature branch until it's merged, but we can also merge when there's a bit more to merge and then you guys can review the bigger pull request.

File: Marlin/Marlin_main.cpp (hunk 1 above)
Comment (KiteLab) [inline Marlin/Marlin_main.cpp]: Changed indentation.

**Comment by daid** [comment]
A pull request should be done on the master branch, to prevent cherry-picking (the master branch is merged towards the UM2go branch, not the other way around)

**Comment by ErikDeBruijn** [comment]
@daid that makes a lot of sense. I'll close this one.

**Pull request closed without merging**